README.md

```diff
@@ -88,15 +88,21 @@ Old listfiles are saved to `logs/{discussion|speedy}.txt.<timestamp>`.
 * Add its dbname to `wikis-enabled` in Git.
 * Make sure that all messages are localized by running `check-i18n.py`.
 * Make sure that the bot account is present on the wiki.
+* Deploy by simply pulling in the new code (the task is continually ran by cron, so you shouldn't need to restart anything):
+  * `become community-tech-tools`
+  * `become commtech-commons`
+  * `cd bot`
+  * `git pull`
+  * The next time the cron rnus, it should include the newly added wiki
```

Comment:
Just a small typo 'rnus' should be 'runs'

Reply:
Fixed!